New self-curated Lahmacun show: LahmaLoudClouds

gammaw opened the pull request:
Please review the addition of our new, reborn "off air premiere" show.

pvj reviewed (approved):
Thank you!

tuz666 reviewed (approved):
Thanks for this, I updated the Show's start & end time from Arcsi's side

gammaw commented:
> Thanks for this, I updated the Show's start & end time from Arcsi's side

@tuz666 I did it already. I see that you now updated it as UTC time. I think it's a misunderstanding. We need to have the arcsi times in CET because this is how we display it on the the webpage. The UTC time is only important for the streaming server.
